Components/Terrain/include/OgreTerrain.i

```diff
@@ -24,8 +24,12 @@
 %include "OgreTerrainMaterialGenerator.h"
 %include "OgreTerrainLayerBlendMap.h"
 %include "OgreTerrainQuadTreeNode.h"
+
+%template(LayerInstanceList) std::vector<Ogre::Terrain::LayerInstance>;
+%template(Pair) std::pair<bool, Ogre::Vector3>;
```

Comment:
Need to find a better name to Pair, as there are another Pairs in the code that aren't <bool, Vector3>